src/web/lib/contributions.ts

```diff
@@ -112,5 +117,35 @@ export const shouldHideSupportMessaging = (
     isRecurringContributor(isSignedIn) ||
     isRecentOneOffContributor();
 
+const REQUIRED_CONSENTS_FOR_ARTICLE_COUNT = [1, 3, 7];
```

Comment:
not the fault of this PR, but this is quite cryptic!

Reply:
it really is!